### ISSUE TITLE
peer reviews copy and pasted rather than uploaded on the ojs

### ISSUE THREAD

**Issue by srees95** [NSAMR/uk.ac.nsamr.journal#413]

**Comment by DeckOfPandas** [comment]
For publication?

If you create a file with all reviewer correspondence rounds, save it as a pdf them add it to the article Galley

**Comment by DeckOfPandas** [comment]
For publication?

If you create a file with all reviewer correspondence rounds, save it as a pdf them add it to the article Galley

**Comment by DeckOfPandas** [comment]
I don't think we should do this. I think the peer review rounds conversations should be added as galley items to a published manuscript.

Dissenting views encouraged!!!
@mhvbyrne
